crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -26,7 +26,7 @@ def initialize(backup)
       end
 
       def restore
-        [:crowbar, :knife, :database].each do |component|
+        [:restore_crowbar, :run_installer, :restore_chef, :restore_database].each do |component|
```

Comment:
Should we rename this method to have no name, then? ;-) Note that I don't care too much -- I prefer the new naming, and don't understand what's good in the old one but will live with anything.